mentoring/mentoring.py

```diff
@@ -104,61 +306,282 @@ def next_step_url(self):
         """
         return '/jump_to_id/{}'.format(self.next_step)
```

Comment:
[The LMS handles URLs like that](https://github.com/edx/edx-platform/blob/16e63c9e/lms/urls.py#L243). AFAIK, they don't work in Studio or any other context besides the LMS.